step-release-vis/src/app/components/environment/environment.ts

```diff
@@ -214,4 +213,44 @@ export class EnvironmentComponent implements OnInit, OnChanges {
     }
     return 'middle';
   }
+
+  /*
+   * Computes the closes snapshot to the current position of the mouse.
+   *
+   * @param svgMouseX the position of the mouse relative to the svg
+   */
+  getSnapshot(svgMouseX: number): void {
```

Comment:
Since the return type is void, maybe rename to updateCurrentSnapshot

Reply:
updated